pkg/injector/patch.go

```diff
@@ -95,3 +105,19 @@ func makePatches(req *admissionv1.AdmissionRequest, pod *corev1.Pod) []jsonpatch
 	admissionResponse := admission.PatchResponseFromRaw(original, current)
 	return admissionResponse.Patches
 }
+
+func mergePortExclusionLists(podOutboundPortExclusionList, globalOutboundPortExclusionList []string) []string {
+	portExclusionListMap := make(map[string]bool)
```

Comment:
Let's use `mapset` here for consistency, and append to the final slice only if `set.Add(element)` returns true, which is the case when the element doesn't already exist in the slice.